manifests/zone.pp

```diff
@@ -102,14 +103,26 @@
         }
 
         if member(['init', 'managed'], $zone_file_mode) {
-            file { "${cachedir}/${name}/${zone_file}":
-                ensure  => present,
-                owner   => $bind_user,
-                group   => $bind_group,
-                mode    => '0644',
-                replace => ($zone_file_mode == 'managed'),
-                source  => pick($source, 'puppet:///modules/bind/db.empty'),
-                audit   => [ content ],
+            if ! empty($content) {
+                file { "${cachedir}/${name}/${zone_file}":
+                  ensure  => present,
+                  owner   => $bind_user,
+                  group   => $bind_group,
+                  mode    => '0644',
+                  replace => ($zone_file_mode == 'managed'),
+                  content => $content,
+                  audit   => [ content ],
```

Comment:
You'll need to get rid of the audit parameters on this and the other file resource. They've been removed on master, and that's the source of the merge conflict.